specification/common-types/resource-management/v1/types.json

```diff
@@ -392,7 +392,7 @@
             "modelAsString": true
           }
         },
-        "lastModifiedByAt": {
+        "lastModifiedAt": {
```

Comment:
This fix is because it doesn't match RPC. https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-contracts.md#system-metadata-for-all-azure-resources